package.json

```diff
@@ -1,142 +1,5 @@
 {
-  "name": "jss",
-  "description": "A lib for generating Style Sheets with JavaScript.",
-  "version": "9.8.1",
-  "author": {
-    "name": "Oleg Slobodskoi",
-    "email": "oleg008@gmail.com"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:cssinjs/jss.git"
-  },
-  "keywords": [
-    "jss",
-    "style",
-    "sheet",
-    "stylesheet",
-    "css",
-    "components",
-    "composable",
-    "css in js",
-    "css-in-js"
-  ],
-  "license": "MIT",
-  "main": "./lib/index.js",
-  "engines": {
-    "node": ">=4"
-  },
-  "devDependencies": {
```

Comment:
All development stuff should stay here. It's cleaner when packages have only production dependencies.